src/tools/namespace_tool.cpp

```diff
@@ -129,8 +138,8 @@ void NameSpaceTool::PrintHelp(const std::string &cmd) {
     } else if (cmd == kSegInfoCmd) {
         std::cout << "curve_ops_tool " << cmd << " -fileName=/test [-mdsAddr=127.0.0.1:6666] [-confPath=/etc/curve/tools.conf]" << std::endl;  // NOLINT
     } else if (cmd == kCleanRecycleCmd) {
-        std::cout << "curve_ops_tool " << cmd << " [-fileName=/cinder] [-mdsAddr=127.0.0.1:6666] [-confPath=/etc/curve/tools.conf]" << std::endl;  // NOLINT
-        std::cout << "If -fileName is specified, delete the files in recyclebin that the original directory is fileName" << std::endl;  // NOLINT
+        std::cout << "curve_ops_tool " << cmd << " [-fileName=/cinder] [-expireTime=1(s|m|h|d|M|y)] [-mdsAddr=127.0.0.1:6666] [-confPath=/etc/curve/tools.conf]" << std::endl;  // NOLINT
```

Comment:
The default expireTime is 3 d. Maybe 'M' and 'y' is not need.

Reply:
> The default expireTime is 3 d. Maybe 'M' and 'y' is not need.

It's a usage message, we should tell others which time units we support.

Reply:
print the meaning of s|m|h|d|M|y

Reply:
> print the meaning of s|m|h|d|M|y

done.